doc/whats_new/v1.3.rst

```diff
@@ -405,7 +411,7 @@ Changelog
 
 - |API| The `eps` parameter of the :func:`log_loss` has been deprecated and will be
   removed in 1.5. :pr:`25299` by :user:`Omar Salman <OmarManzoor>`.
-  
+
 - |Feature| :func:`metrics.average_precision_score` now supports the
```

Comment:
Could you fix the formatting here to avoid a diff

Reply:
I think this can't be helped, a hook made this modification (I forgot, maybe it's `black`?) during my pre-commit.

Reply:
Indeed it looks like it must have been incorrectly retained in another PR. All good, thanks!